### PR TITLE
Prevent crash when XInput is not present

### PR DESCRIPTION
**Pull request by winterhell**
Now returns from the function instead of throwing an exception

### CODE REVIEW

**Comment by VPeruS** [comment]
Why not flag that user can check?

**Comment by winterhell** [comment]
Can you do that? I don't know how to document it.
Plus so far nobody ever publicly cared that it crashes, I might be the only one.

**Comment by VPeruS** [comment]
Nevermind, it is very platform specific things and it should be hidden in joystick handler(like with XI2, XKB for linux platform) and user shouldn't know about it. I thought Frassle even create pull request where he change exception message coz he think it will be related to this. Also it very hard to get same platfrom setup to test.

File: src/OpenTK/Platform/Windows/XInputJoystick.cs [hunk 1]
```diff
@@ -381,7 +381,8 @@ internal XInput()
                 }
                 if (dll == IntPtr.Zero)
                 {
-                    throw new NotSupportedException("XInput was not found on this platform");
+                  //XInput was not found on this platform
```
Comment (Frassle) [inline src/OpenTK/Platform/Windows/XInputJoystick.cs]: Please leave a Debug.Print in with the old exception message

**Comment by varon** [comment]
Thanks, @winterhell  - It's really great to see external contributions like this!

**Pull request merged**